### PR TITLE
Ph calibration: midpoint, lowpoint, highpoint

### PR DESCRIPTION
**Pull request by eucalvo**

### CODE REVIEW

File: src/UIState/MainMenu.cpp [hunk 1]
```diff
@@ -170,15 +170,12 @@ void MainMenu::selectView() {
...
-      break;
-    case SET_CALIBRATION_2:
-      this->setNextState((UIState *)new PHCalibration(tc));
-      break;
-    case SET_CALIBRATION_CLEAR:
-      this->setNextState((UIState *)new PHCalibration(tc));
+    case SET_CALIBRATION:
+      this->setNextState((UIState *)new PHCalibrationMid(tc));
       break;
+    // case SET_CALIBRATION_CLEAR:
```
Comment (jgfoster) [inline src/UIState/MainMenu.cpp:176]: Since this is part of MainMenu we should probably implement at least a stub that does nothing.

**Review by jgfoster — CHANGES_REQUESTED** [review]
Just a couple lines of commented-out code (as well as whatever spelling or other tests remain).

**Comment by jgfoster** [comment]
Rather than using `sprint()` we should use `snprintf()`. See other examples in the code.

File: src/UIState/MainMenu.cpp (hunk 1 above)
Comment (jgfoster) [reply src/UIState/MainMenu.cpp:176]: See #96.

**Review by jgfoster — APPROVED** [review]
I've addressed the issues I raised.

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**